jscomp/test/keep_uncurry_attribute.js

```diff
@@ -0,0 +1,16 @@
+// Generated by ReScript, PLEASE EDIT WITH CARE
+'use strict';
+
+let Curry = require("../../lib/js/curry.js");
+
+function map1(prim0, prim1) {
+  return map(prim0, Curry.__1(prim1));
```

Comment:
It's puzzling that the `@uncurry` attribute has the opposite effect of what's supposed to do on this example, but that's the reason for this PR.